spec/CookiePluginSpec.php

```diff
@@ -146,7 +146,12 @@ function it_saves_cookie(RequestInterface $request, ResponseInterface $response,
 
         $response->hasHeader('Set-Cookie')->willReturn(true);
         $response->getHeader('Set-Cookie')->willReturn([
-            'cookie=value',
+            sprintf(
+                'cookie=value; expires=%s; Max-Age=60; path=/; domain=test.com; secure; HttpOnly',
+                (new \DateTime())->add(
+                    new \DateInterval('PT60S')
```

Comment:
that sure would be simpler. lets put a fixed date 10 years in the future
so it looks not weird with an already expired date :-D